components/molecules/LikeStat/LikeStat.tsx

```diff
@@ -44,7 +46,7 @@ const LikeStat: React.FC<LikeStatProps> = (props) => {
   return (
     <View style={styles.container}>
       <Image
-        source={require('./like.png')}
+        source={require('./like@3x.png')}
```

Comment:
@J-Rehan Did you mean to explicitly reference @3x? I thought the way responsive images worked in react native was that you always just reference the original file name and it adds @2x or @3x dynamically based on the device? I've never actually verified that though.

@nbonatsakis do you know, for sure?